components/engine_rocks/src/encryption.rs

```diff
@@ -12,7 +12,7 @@ use rocksdb::{
 use crate::{r2e, raw::Env};
 
 // Use engine::Env directly since Env is not abstracted.
-pub(crate) fn get_env(
```

Comment:
Exposed to parse SST files.

Reply:
You should use `engine_rocks::get_env` instead.

Reply:
fixed